src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -286,7 +282,7 @@ private String getQuestionResultsStatisticsHtmlQuestionView(List<FeedbackRespons
                 incomingPoints[i] = teamResult.normalizedPeerContributionRatio[i][studentIndx];
             }
             contribFragments.append(Templates.populateTemplate(
-                    FeedbackQuestionFormTemplates.CONTRIB_RESULT_STATS_FRAGMENT,
+                    FormTemplates.CONTRIB_RESULT_STATS_FRAGMENT,
                     "${studentTeam}", Sanitizer.sanitizeForHtml(displayTeam),
                     "${studentName}", Sanitizer.sanitizeForHtml(displayName),
                     "${CC}", getPointsAsColorizedHtml(summary.claimedToInstructor),
```

Comment:
It will be nice to use SLOTS for all `${  }` eventually, if not in this PR. Unless there is a drawback to doing that.